main.go

```diff
@@ -288,16 +288,13 @@ func IntroCmd() error {
 	}
 	clientLog.Output()
 	clientLog.Output()
-	clientLog.Output(coreutils.PrintTitle("Thank you for installing JFrog CLI! 🐸"))
+	clientLog.Output(coreutils.PrintTitle("				Thank you for installing JFrog CLI! 🐸 \n				Here's how you get started using JFrog CLI.\n"))
```

Comment:
Let's add the installed JFrog CLI version as follows -
Thank you for installing version x.y.x of JFrog CLI! 🐸

Here's how you get started using JFrog CLI.
-->
Here's how you get started.